web/src/features/AppVersionHistory/AppVersionHistoryRow.jsx

```diff
@@ -775,7 +776,21 @@ class AppVersionHistoryRow extends Component {
                 <span className="u-fontWeight--bold">{releasedTs}</span>
               </p>
             )}
-            {this.renderDiff(version)}
+            {this.state.showViewDiffButton && (
+              <ViewDiffButton
```

Comment:
this is the component extracted from `<AppVersionHistory>`

Reply:
nice